examples/kubernetes/kuma-demo/kuma-demo-log.yaml

```diff
@@ -0,0 +1,103 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: logging
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: logstash
+  namespace: logging
+spec:
+  selector:
+    app: logstash
+  ports:
+  - name: tcp
+    port: 5000
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: logstash-config
+  namespace: logging
+data:
+  logstash.conf: |
+    input {
+      tcp {
+          port => 5000
+          codec => json
+      }
+    }
+    output{
+      loggly{
+        key => "dbee0a24-abf3-4049-a50a-8c80531a9e11"
```

Comment:
Make sure that `https://kumademo.loggly.com/` is still working (we only had a Trial license for 14 days)